### PR TITLE
Add test for overwriting wrapper

### PR DESCRIPTION
**Pull request by oehme**

### CODE REVIEW

File: subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperGenerationIntegrationTest.groovy [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 package org.gradle.integtests
 
+
```
Comment (wolfs) [inline subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperGenerationIntegrationTest.groovy]: Nit: additional newline.

File: subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperGenerationIntegrationTest.groovy [hunk 2]
```diff
@@ -80,6 +81,14 @@ class WrapperGenerationIntegrationTest extends AbstractIntegrationSpec {
         file("first/gradlew.bat").md5Hash == file("second/gradlew.bat").md5Hash
     }
 
+    def "can successfully run the wrapper task twice"() {
+        when:
+        succeeds("wrapper", "--gradle-version=5.0")
```
Comment (wolfs) [inline subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperGenerationIntegrationTest.groovy]: Why are we passing the version here? It should fail as well when we create a wrapper for the current Gradle version, right?

Comment (oehme) [reply subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperGenerationIntegrationTest.groovy]: If I don't specify a version it'll create a URL that doesn't exist yet. I'll think of something else.

Comment (big-guy) [reply subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperGenerationIntegrationTest.groovy]: If you use `AbstractWrapperIntegrationSpec`, you should be able to use `prepareWrapper` to generate a URL to a local bin distribution and the `wrapperExecutor` to run the build.

Comment (oehme) [reply subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperGenerationIntegrationTest.groovy]: Thanks, that's exactly what I need.

**Comment by oehme** [comment]
@wolfs Please have another look

**Comment by oehme** [comment]
Yep: https://builds.gradle.org/viewLog.html?buildId=18626815&tab=queuedBuildOverviewTab

**Pull request merged**